wrapper-generator/src/test/kotlin/it/krzeminski/githubactions/wrappergenerator/generation/GenerationTest.kt

```diff
@@ -512,6 +512,57 @@ class GenerationTest : FunSpec({
         )
     }
 
+    test("action v2 deprecated by v3") {
+        // given
+        val actionManifestHasNoInputs = emptyMap<String, Input>()
+        val actionManifest = Metadata(
+            inputs = actionManifestHasNoInputs,
+            name = "Action With No Inputs",
+            description = "Description",
+        )
+
+        val coords = ActionCoords("john-smith", "action-with-no-inputs", "v2", deprecatedByVersion = "v3")
```

Comment:
Let's use action's name that corresponds to test's subject, e.g. here
```suggestion
        val coords = ActionCoords("john-smith", "deprecated-action "v2", deprecatedByVersion = "v3")
```
and also see the target file name.